security/s2a/crypter/record.go

```diff
@@ -0,0 +1,16 @@
+package crypter
+
+// S2AAeadCrypter is the interface for gRPC S2A record protocol.
```

Comment:
Could we change this comment to `is the interface for an AEAD cipher used by the S2A record protocol.`?